baseplate/lib/events.py

```diff
@@ -93,10 +94,15 @@ class EventQueue(ContextFactory, config.Parser, Generic[T]):
 
     """
 
-    def __init__(self, name: str, event_serializer: Callable[[T], bytes]):
-        self.queue = MessageQueue(
-            "/events-" + name, max_messages=MAX_QUEUE_SIZE, max_message_size=MAX_EVENT_SIZE
-        )
+    def __init__(
+        self, name: str, event_serializer: Callable[[T], bytes], use_in_memory_queue: bool = False
```

Comment:
Again, I think Freds suggestion of a queue type value would be good here. You could even define an [enum](https://docs.python.org/3/library/enum.html?highlight=enum#module-enum) if you want to up the robustness and type safety.